core/src/main/scala/chisel3/When.scala

```diff
@@ -60,6 +60,20 @@ object when {
   }
 }
 
+private object Scope {
+  sealed trait Type
+
+  /** Indicates that the `WhenContext` is in the "if" clause. */
+  object If extends Type {
+    override def toString: String = "if"
+  }
+
+  /** Indicatest that the `WhenContext` is in the "else" clause. */
```

Comment:
"Indicatest"

Reply:
I love that, "indicate by testing"

Reply:
Fixed in https://github.com/chipsalliance/chisel/pull/4184/commits/4b1c8f1d811cf8c5897c33b9510e6bb063e499e9.